package-testing/scenarios.json

```diff
@@ -1,14 +1,15 @@
 {
   "scenarios": {
-    "baseUFC": {
-      "ufcPath": "ufc/flags-v1.json",
-      "banditModelPath": null,
-      "testCases": "ufc/tests"
+
+    "jitBandits1": {
```

Comment:
scenario 1: cold_start_bandit has modelVersion "cold start"
scenario 2: two bandits have the modelVersion "cold start"